charts/risingwave/values.yaml

```
@@ -495,6 +495,11 @@ stateStore:
     ##
     bucket: ""
 
+    ## @param stateStore.s3.isForcePathStyle S3 path format. 
+    ## env 'RW_IS_FORCE_PATH_STYLE' is used
```

Link the descriptions of the two styles. Remove the mention of the env because it's meaningless to the chart user.

```suggestion
    ## @param stateStore.s3.isForcePathStyle Enforce path style requests. If the value is false, the path could be one
    ## of path style and virtual hosted style, depending on the endpoint format. For more details of the two styles, 
    ## please refer to the documentation: https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html
```